js/prestashop-ui-kit.js

```diff
@@ -3,6 +3,7 @@ import 'bootstrap';
 import 'select2';
 import 'jquery.growl';
 import 'pstagger';
+var initInputFileChangeEvent = require('./events/file-input.js');
```

Comment:
Use `import` instead or require.

Also, maybe it would be better to have `/components/file-input` instead of `/events/file-input`.